container/sagemaker/serve.py

```diff
@@ -226,13 +227,19 @@ def _setup_gunicorn(self):
     def _create_nginx_config(self):
         template = self._read_nginx_template()
         pattern = re.compile(r'%(\w+)%')
+        ping_request = JS_PING
```

Comment:
we could actually check for tfs 1.11 here as well, and assign the ping_tfs_1_11 handler. that would get rid of a request-time if statement in the javascript ping handler.

Reply:
Merged 1.11 ping check and mme ping check together.